structure_tests/Makefile

```diff
@@ -12,22 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION := 1.0.0
+# Bump these on release
+VERSION_MAJOR ?= 0
+VERSION_MINOR ?= 1
```

Comment:
looks like we're going to 0.1.0 from 1.0.0?

Reply:
well we never released a 1.0.0. this Makefile never got used before